spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/outbound/AbstractMqttMessageHandler.java

```diff
@@ -134,7 +134,7 @@ public boolean isRunning() {
 
 	@Override
 	protected void handleMessageInternal(Message<?> message) throws Exception {
-		this.connectIfNeeded();
+		connectIfNeeded();
```

Comment:
According to the comment on this **deprecated** method, its usage should be removed from here.
If it's only concern, I'll fix on merge.